android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -309,8 +296,8 @@ public RNBranchModule(ReactApplicationContext reactContext) {
         return constants;
     }
 
-    private void forwardInitSessionFinishedEventToReactNative(ReactApplicationContext reactContext) {
-        mInitSessionEventReceiver = new BroadcastReceiver() {
+    private void forwardInitSessionEventsToReactNative(ReactApplicationContext reactContext) {
```

Comment:
Since we are renaming the method anyway, could we rename it to `setupForwardingInitSessionEventsToReactNative`? (This name wouldn't imply an action and IMHO would reflect what the method does better.)

Reply:
Good suggestion. I agree.